docs/join2019_submisson.md

```diff
@@ -0,0 +1,31 @@
+#### Session Title
+
+### A Reliable Development Workflow with CI for Looker
+
+#### Session Abstract
+
+Occasionally, when we make changes to LookML or transform an underlying database table, we break downstream experiences in Looker by introducing database or content errors. These bugs are easy to deploy to production unknowingly, since they are runtime errors, not compiler errors. In short, they aren't discovered until a user runs a query. For little-used fields, these bugs can remain hidden for weeks or months.
+
+We believe in the power of CI for analytics, firmly of the view that analytics can adopt best practices from software engineering. We wanted a tool that would check for fields with database errors and content with LookML errors. We wanted to leverage continuous integration (CI) pipelines as part of LookML deployments or proposed database changes to provide a seamless development experience. Ultimately, we wanted to establish a baseline performance expectation for our Looker instance. 
```

Comment:
I've altered this slightly, making the second paragraph more explicitly about the values that drove the development of the tool. Let me know if you think that works. Happy to revert.